src/riak_kv_quanta.erl

```diff
@@ -0,0 +1,250 @@
+-module(riak_kv_quanta).
+
+-export([quanta/3,
+         timestamp_to_ms/1,
+         ms_to_timestamp/1]).
+
+%% @doc This module serves to generate time quanta on multi - (year, month, day, hour, minute,
```

Comment:
You aren't sorry ;). interesting about the type docs. I suggest people read
the code lol. I will see what's up, although I've never run edoc.

-Andrew

On Tue, Mar 17, 2015 at 1:16 PM, Sean Cribbs <notifications@github.com>
wrote:

> In src/riak_kv_quanta.erl
> <https://github.com/basho/riak_kv/pull/1092#discussion_r26593759>:
>
> > @@ -0,0 +1,250 @@
> > +-module(riak_kv_quanta).
> > +
> > +-export([quanta/3,
> > +         timestamp_to_ms/1,
> > +         ms_to_timestamp/1]).
> > +
> > +%% @doc This module serves to generate time quanta on multi - (year, month, day, hour, minute,
>
> Sorry to be pedantic - this needs to be above the module declaration.
> Also, you should probably verify that the type docs come out as expected,
> I've found them to be fiddly in the past (such that I don't remember how to
> do them correctly).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/basho/riak_kv/pull/1092/files#r26593759>.
>